client/src/components/common/Form.js

```diff
@@ -0,0 +1,54 @@
+import React, { createRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form as ReactstrapForm, Input, Col, Button, FormGroup, Label,
```

Comment:
put in separate lines please.. the imports

Reply:
done